inc/Subscriptions/BulkSubscriptionSender.php

```diff
@@ -222,18 +228,25 @@ protected function sendDigest($subscriber_mail, $id, $lastupdate)
      * @author Adrian Lang <lang@cosmocode.de>
      *
      */
-    protected function sendList($subscriber_mail, $ids, $ns_id)
+    protected function sendList($subscriber_mail, $ids, $ns_id, $lastupdate)
     {
+        global $lang;
+
         if ($ids === []) {
             return false;
         }
 
         $tlist = '';
         $hlist = '<ul>';
         foreach ($ids as $id) {
+            $last = $this->lastRevBefore($id, $lastupdate);
```

Comment:
If I understand the method above correctly, it might return null, correct? There should probably not be a diff link further down then?